### ISSUE TITLE
Typo: on page https://docs.scipy.org/doc/scipy/reference/generated/scipy.integrate.dblquad.html#scipy.integrate.dblquad

### ISSUE THREAD

**Issue by davidjabon** [scipy/scipy#17450]
### Describe your issue.

There is a typo on the page https://docs.scipy.org/doc/scipy/reference/generated/scipy.integrate.dblquad.html#scipy.integrate.dblquad
On the very last example, the code examples integrate from x to 2-x, but the displayed integral integrates from 2-x to x.

![typo_on_dblquad](https://user-images.githubusercontent.com/12072431/202865288-bca0c976-a2c3-4698-8fe6-8a113d3d6fd7.PNG)

### Reproducing Code Example

```python
NA
```

### Error message

```shell
NA
```

### SciPy/NumPy/Python version information

1.9.3

**Comment by j-bowhay** [comment]
Would you like to submit a pr @davidjabon?

**Comment by boatwrong** [comment]
@j-bowhay If it is okay I'd be interested in submitting a pr for this.

**Comment by j-bowhay** [comment]
@boatwrong yes go for it. As it is a documentation only fix please try to skip the appropriate CI jobs: https://scipy.github.io/devdocs/dev/contributor/continuous_integration.html#skipping
